assembly/src/test_util.rs

```diff
@@ -32,11 +32,15 @@ pub(crate) fn code_to_prom(code: &[crate::Instruction]) -> crate::ProgramRom {
     let mut prom = crate::ProgramRom::new();
     // TODO: type-gate field_pc and use some `incr()` method to abstract away `+1` /
     // `*G`.
-    let mut pc = B32::ONE; // we start at PC = 1G.
+    let mut pc = B32::ONE; // we start at PC = 0G.
     for &instruction in code.iter() {
-        let interp_inst = InterpreterInstruction::new(instruction, pc, None, false);
+        let prover_only = instruction[0] == Opcode::Alloci.get_field_elt();
+        let interp_inst = InterpreterInstruction::new(instruction, pc, None, prover_only);
```

Comment:
It's working for now, but as discussed offline I'd rather have it already generic.